docs_user/modules/con_adoption-limitations.adoc

```diff
@@ -2,17 +2,48 @@
 
 = Adoption limitations
 
+Before you proceed with the adoption, check which features are considered a Technology Preview or are unsupported.
+
+.Technology Preview
+
+The following features are considered a Technology Preview and have not been tested within the context of the {rhos_long_noacro} adoption:
+
+* {bare_metal_first_ref}
+* NFS Ganesha back end for {rhos_component_storage_file_first_ref}
+* iSCSI, NFS, and FC-based drivers for {block_storage_first_ref}
+* The following {compute_service_first_ref} features:
+** Compute hosts with `/var/lib/nova/instances` on NFS
+** Hugepages
```

Comment:
Afer chatting to @eshulman2 we're satisfied that NFV have MVP testing of huge pages, and we can declare it supported.